src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.props

```diff
@@ -70,6 +70,13 @@
         <TargetPath>runtimes/$(RuntimeIdentifier)/native/include/%(RecursiveDir)</TargetPath>
       </RuntimeFiles>
 
+      <!-- FIXME: Can this condition be '$(RuntimeFlavor)' == 'mono' and not TargetsMobile? We will want this for desktop mono runtime packs too -->
+      <RuntimeFiles Condition="'$(TargetsMobile)' == 'true'"
```

Comment:
You should be able to rely on '$(RuntimeFlavor)' == 'mono' for all our configurations